pkg/config/options.go

```diff
@@ -24,14 +25,19 @@ type DbOptions struct {
 	WaitForDB              time.Duration `long:"wait-for"                 description:"wait for DB on start, up to specified duration"`
 }
 
+type PayerOptions struct {
+	PrivateKey string `long:"private-key" description:"Private key used to sign blockchain transactions"`
+}
+
 type ServerOptions struct {
 	LogLevel string `short:"l" long:"log-level"    description:"Define the logging level, supported strings are: DEBUG, INFO, WARN, ERROR, DPANIC, PANIC, FATAL, and their lower-case forms." default:"INFO"`
 	//nolint:staticcheck
 	LogEncoding string `          long:"log-encoding" description:"Log encoding format. Either console or json"                                                                                  default:"console" choice:"console"`
 
-	PrivateKeyString string `long:"private-key" description:"Private key to use for the node"`
+	SigningKeyString string `long:"signing-key" description:"Signing key used to sign messages"`
```

Comment:
Would 'signer-private-key' make sense, for consistency purposes?

Reply:
Sure. Will change.